audit/sink_metric_labeler.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package audit
+
+import (
+	"strings"
+
+	"github.com/hashicorp/eventlogger"
+	"github.com/hashicorp/vault/internal/observability/event"
+)
+
+const (
+	metricLabelAuditSinkSuccess         = "audit.sink.success"
```

Comment:
rather than calling `splitLabel` every time `Labels()` is called, can we define the labels as slices? e.g. 

```
var (
  metricLabelAuditSinkSuccess = []string{"audit", "sink", "success"}
  ...
)
```

Reply:
I went with `const` to make sure nothing meddled with the value, but I guess it's not exported so maybe less risk.  👍🏼